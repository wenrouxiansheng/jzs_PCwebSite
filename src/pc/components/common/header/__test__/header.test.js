import React from "react";
import renderer from "react-test-renderer";
import Header from "../index";

describe("PC:Header", () => {
  it("渲染测试", () => {
    const component = renderer.create(<Header />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
    //渲染测试
  });
});
