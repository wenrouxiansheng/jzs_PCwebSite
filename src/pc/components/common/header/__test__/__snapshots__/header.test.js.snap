// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PC:Header 渲染测试 1`] = `
<header
  className="headModule"
>
  <img
    alt=""
    src="test-file-stub"
  />
  <div
    className="headerRight"
  >
    <span
      className="city"
    >
      北京
    </span>
    <img
      alt=""
      src="test-file-stub"
    />
    <span
      className="tel"
    >
      400-900-8898
    </span>
  </div>
</header>
`;
